### ISSUE TITLE
solution does not follow instructions

### ISSUE THREAD

**Issue by williamhancharek** [learn-co-students/js-jquery-event-listeners-readme-bootcamp-prep-000#402]
the instructions asked for the functions to be defined outside of document.ready(function {} ) and for the functions to be called inside of document.ready(function {} ).  However the solutions provided do not make any use of the space inside document.ready(function {} )

**Comment by josephclander** [comment]
I agree. I noted the instruction but forgot to call them within the document ready function and it still passed.
